### PR TITLE
docs: add browser support page [SFUI2-1161]

### PR DESCRIPTION
**Pull request by Szymon-dziewonski**
# Related issue

https://vsf.atlassian.net/browse/SFUI2-1161

# Scope of work

Adding browser support page in documentation

# Screenshots of visual changes

image

# Checklist

- [x] Self code-reviewed
- [ ] Changes documented
- [ ] Semantic HTML
- [ ] SSR-friendly
- [ ] Caching friendly
- [ ] a11y for WCAG 2.0 AA
- [ ] examples created
- [ ] blocks created
- [ ] cypress tests created

### CODE REVIEW

File: apps/docs/components/react/browser-support.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+layout: DefaultLayout
+hideBreadcrumbs: true
+---
+
+# Browser support
+
+This documentation provides information about the browser support for Storefront-ui, a powerful and versatile front-end UI framework. Understanding the browser compatibility of a UI framework is crucial for developers to ensure that their applications work seamlessly across different browsers and versions. This document will outline the supported browsers, their versions, and any specific considerations or known issues related to compatibility.
```
Comment (mattmaribojoc) [inline apps/docs/components/react/browser-support.md:8]:
```suggestion
Understanding the browser compatibility of a UI framework is crucial for developers to ensure that their applications work seamlessly across different browsers and versions. This document will outline the supported browsers, their versions, and any specific considerations or known issues related to compatibility.
```

I think the first sentence is unnecessary - we can assume that someone knows what Storefront UI is if they're on our docs, and I think just getting right into the document is better than describing it.

File: apps/docs/components/react/browser-support.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+## The following browsers are officially supported by Storefront-ui:
+
+**2 last stable versions of each browser**
```
Comment (mattmaribojoc) [inline apps/docs/components/react/browser-support.md:12]:
```suggestion
**The 2 latest stable versions of:**
```

I think this better sets up the list, but feel free to ignore.

File: apps/docs/components/react/browser-support.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+- Chrome
+- Chrome for Android
+- Firefox
+- Edge
+- Safari
+- Safari on IOS
+- Opera
+
+**latest Firefox ESR (Extended Support Release) [Firefox Release Calendar](https://wiki.mozilla.org/index.php?title=Release_Management/Calendar&redirect=no)**
```
Comment (mattmaribojoc) [inline apps/docs/components/react/browser-support.md:22]:
```suggestion
**The latest Firefox ESR (Extended Support Release) [Firefox Release Calendar](https://wiki.mozilla.org/index.php?title=Release_Management/Calendar&redirect=no)**
```

Comment (Szymon-dziewonski) [reply apps/docs/components/react/browser-support.md]: https://github.com/vuestorefront/storefront-ui/pull/2883/commits/d9f815bdedbf9008a218a932bd4aa3c303c5f653

File: apps/docs/components/react/browser-support.md (hunk 2 above)
Comment (Szymon-dziewonski) [reply apps/docs/components/react/browser-support.md]: Good suggestion, here's change https://github.com/vuestorefront/storefront-ui/pull/2883/commits/d9f815bdedbf9008a218a932bd4aa3c303c5f653

File: apps/docs/components/react/browser-support.md (hunk 1 above)
Comment (Szymon-dziewonski) [reply apps/docs/components/react/browser-support.md]: I agree, thanks https://github.com/vuestorefront/storefront-ui/pull/2883/commits/d9f815bdedbf9008a218a932bd4aa3c303c5f653

**Review by mattmaribojoc — APPROVED** [review]
Great work!

**Pull request merged**